code/datums/traits/negative.dm

```diff
@@ -579,3 +579,43 @@
 	lose_text = "<span class='notice'>Your mind finally feels calm.</span>"
 	medical_record_text = "Patient's mind is in a vulnerable state, and cannot recover from traumatic events."
 	hardcore_value = 9
+
+/datum/quirk/allergic
+	name = "Extreme Medicine Allergy"
+	desc = "Ever since you were a kid you always were allergic to certain chemicals..."
+	value = -2
+	gain_text = "<span class='danger'>You feel your immune system shift.</span>"
+	lose_text = "<span class='notice'>You fell your immune system phase back into perfect shape.</span>"
```

Comment:
it's 5pm honey, time for your daily grammar check

this should probably be "feel" rather than "fell"

Reply:
it's 5pm honey, time for your daily "use the suggestions tool for grammar review" whine